### PR TITLE
[SPARK-17338][SQL][follow-up] add global temp view

### PR DESCRIPTION
**Pull request by cloud-fan**
## What changes were proposed in this pull request?

address post hoc review comments for https://github.com/apache/spark/pull/14897
## How was this patch tested?

N/A

### CODE REVIEW

**Comment by cloud-fan** [comment]
cc @rxin @yhuai

File: sql/core/src/main/scala/org/apache/spark/sql/catalog/Catalog.scala [hunk 1]
```diff
@@ -270,9 +270,10 @@ abstract class Catalog {
    * tied to any databases, i.e. we can't use `db1.view1` to reference a local temporary view.
    *
```
Comment (rxin) [inline sql/core/src/main/scala/org/apache/spark/sql/catalog/Catalog.scala]: can you add a line saying the return type was unit in Spark 2.0, but changed to boolean in Spark 2.1?

**Comment by rxin** [comment]
LGTM other than the two minor comments.

We also need a Python API for this, don't we?

**Comment by rxin** [comment]
LGTM pending Jenkins.

**Comment by cloud-fan** [comment]
merging to master!

**Pull request closed without merging**